workspace/apps/pidp/src/app/features/organization-info/pages/endorsements/components/endorsement-card/endorsement-card.component.html

```diff
@@ -1,47 +1,23 @@
 <section class="box" uiPidpInjectViewportCss>
   <div class="content">
     <div class="icon-box">
-      <mat-icon class="material-icons-outlined">account_circle</mat-icon>
+      <img src="/assets/images/icons/account-image.svg" alt="Account Icon" />
     </div>
-    <div class="spacer-left"></div>
     <div class="name-box">
       <h3 class="name">{{ nameText }}</h3>
     </div>
+  </div>
+  <div class="second-row">
     <div class="school-box">
-      <span>{{ collegeText }}</span>
+      <p *ngIf="collegeText"> <span>Pharmacists: {{ collegeText }}</span> </p>
```

Comment:
The text `Pharmacists` is probably just from the figma mockup. In practice we want to only display the name of the college from the licence.